Update README.md (#68)
melontini/bootloader-unlock-wall-of-shame: brands/xiaomi/README.md

- [This source][bootloader-unlock-block-mainland-china] claims that the final day one could use the provided form to request an unlock was September 9, 2024. It also has an update, it seems that Xiaomi is suggesting inside the Community App that they will bring back the service in 2025, but it wouldn't be the first time a company lied about this.
- In various forum posts [like this one][bootloader-unlock-block-mainland-china-alt] it is mentioned that a china region phone can only be unlocked if one is situated in China and has a level 5 Xiaomi account, or pays for the service in the hopes that a technician can unlock it.

> - From 6th of february there are [other restrictions] or requirements, call them as you like, like a 180 days without unban or warnings on the Xiaomi Community Acc. and no recent sensitive changes into the Xiaomi acc, also the unlocking should be done on the same device and with no ip changes. Also seems that users will have to unlock when they can and in time or won't have the possibility to unlock anymore.

Reviewer: based on the wording in the article, you _can_ be flagged for switching ip-addresses, I assume there's a threshold. "no ip changes" should probably be rephrased to reflect that.

Author: Added the suggestion, also fixed a typo for the link and there was a point that should had to be removed and so I removed it. Now it's clean

> - From 6th of february there are [other restrictions] or requirements, call them as you like, like a 180 days without unban or warnings on the Xiaomi Community Acc. and no recent sensitive changes into the Xiaomi acc, also the unlocking should be done on the same device and with no ip changes. Also seems that users will have to unlock when they can and in time or won't have the possibility to unlock anymore.

Reviewer: > restrictions or requirements, call them as you like

just leave it as requirements.

"acc" should be expanded into the full word, and Xiaomi Community is redundant in this case. Also "to the account" might be more legible.

```suggestion
- As of february 6th there are [other requirements], like 180 days without bans or warnings on the account. and no recent sensitive changes to the account, also the unlock should be done on the same device and with no ip changes. Also seems that users will have to unlock when they can and in time or won't have the possibility to unlock anymore.
```

* Your Mi Account has been active for more than 30 days.
* Xiaomi Community App version 5.3.31 or above.
* Xiaomi has [updated their policies][updated-policies], further restricting bootloader unlocking. As of January 1st, 2025, they only let you unlock 1 device per year. *What a good way to start the year, right?*

> * And if this wasn't enoough now in China from 6th of february there are [other restrictions] or requirements, call them as you like.

Reviewer: Wrong section. Should be under China and not Global
